### PR TITLE
Run Link Validator

### PR DESCRIPTION
**Pull request by ennru**
... on PRs and once a week.

Update a few links that redirect.

### CODE REVIEW

File: .github/workflows/link-validator.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  validate-links:
+    runs-on: ubuntu-18.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        # See https://github.com/actions/checkout/issues/299#issuecomment-677674415
+        ref: ${{ github.event.pull_request.head.sha }}
+
+      - name: Run Antora
+        run: make all
```
Comment (raboof) [inline .github/workflows/link-validator.yml:19]: Is this supposed to be here? should it be

```suggestion
        run: sbt paradox
```

**Review by raboof — APPROVED** [review]
nice

File: .github/workflows/link-validator.yml (hunk 1 above)
Comment (ennru) [reply .github/workflows/link-validator.yml]: Thanks.

File: .github/workflows/link-validator.yml [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+        uses: coursier/cache-action@v5
+
+      - name: sbt site
+        run: sbt docs/makeSite
+
+      - name: Install Coursier command line tool
+        run: curl -fLo cs https://git.io/coursier-cli-linux && chmod +x cs && ./cs
+
+      - name: Run Link Validator
+        run: ./cs launch net.runne::site-link-validator:0.2.0 -- scripts/link-validator.conf
```
Comment (raboof) [inline .github/workflows/link-validator.yml:34]: Where do the results of those jobs go?

**Review by raboof — APPROVED** [review]

**Pull request merged**